experimental/python-flask/stack.yaml

```diff
@@ -0,0 +1,13 @@
+name: Python Flask application
+version: 0.2.0
```

Comment:
Please change version from 0.2.0 to 0.1.0

Reply:
see next comment....